app/scripts/lib/sentry.js

```diff
@@ -43,6 +43,22 @@ define(function (require, exports, module) {
       if (data.request.headers && data.request.headers.Referer) {
         data.request.headers.Referer = cleanUpQueryParam(data.request.headers.Referer);
       }
+
+      // in production we remove cache busting file names from js files.
+      // replace any 'scripts/1a22742b.head.js' with 'scripts/head.js'
+      // this helps errors consistent across deploys
+      var removeRegex = /\/[0-9A-Fa-f]{8}./g;
```

Comment:
If you only want a final period to be removed, you should use `\.`, otherwise, any character that follows the hex value will be removed. You can also shorten this by using case insensitive matching:

```js
var removeRegex = /\/[0-9a-f]{8}\./gi;
```